t/variation.t

```diff
@@ -118,7 +118,8 @@ is_json_POST($base,$post_data,$expected_result,"Try to POST list of variations")
         'strand' => 1,
         'coord_system' => 'chromosome',
         'allele_string' => 'C/T',
-        'start' => 25744274
+        'start' => 25744274,
+        'ancestral_allele' => undef,
```

Comment:
is this supposed to be added?

Reply:
I deleted this by mistake. Then I added it according to PR #404 (which I missed before I PR mine). I'm not adding the ancestral_allele, I'm just fixing it.

Reply:
I stand corrected, it's just swapping places in the hash...
this solves the 2 insertions and deletions of `'ancestral_allele' => undef`

Reply:
I stand corrected, it's just swapping places in the hash...
this solves the 2 insertions and deletions of `'ancestral_allele' => undef`